runner/src/instrument.ts

```diff
@@ -0,0 +1,16 @@
+import {config} from "./server/plugins/utils/AdapterConfigurationSchema";
+import * as Sentry from "@sentry/node";
+import {NodeOptions} from "@sentry/node";
+
+if (config.sentryDsn) {
+    const sentryConfig: NodeOptions = {
+        dsn: config.sentryDsn, // Replace with your Sentry DSN
+        environment: config.sentryEnv || "prod" // Use the provided environment or default to "prod"
```

Comment:
I'm not convinced about a default value here being prod - maybe it should be `development`? Or just no default value and we only initialise if both sentryDsn and sentryEnv are set?